linera-service/template/state.rs.template

```diff
@@ -0,0 +1,9 @@
+use linera_sdk::views::{ViewStorageContext, RegisterView};
+use linera_views::views::{GraphQLView, RootView};
+
+#[derive(RootView, GraphQLView)]
+#[view(context = "ViewStorageContext")]
+pub struct Application {
+    pub value: RegisterView<u64>,
```

Comment:
Is this as an illustration for how to add fields, or are empty views not supported?